akka-remote/src/test/scala/akka/remote/artery/SendQueueSpec.scala

```diff
@@ -51,7 +51,8 @@ object SendQueueSpec {
 class SendQueueSpec extends AkkaSpec("""
     akka.actor.serialize-messages = off
     akka.stream.materializer.debug.fuzzing-mode = on
-  """.stripMargin) with ImplicitSender {
+    akka.stream.secret-test-fuzzing-warning-disable = yep
```

Comment:
wow, is `yep` valid?

Reply:
Hehe, this particular setting just checks for any string, not the actual value.